src/main/java/wolox/training/models/Users.java

```diff
@@ -0,0 +1,85 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+import java.util.Vector;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity
+public class Users {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private long id;
+
+  @Column(nullable = false)
+  private String username;
+
+  @Column(nullable = false)
+  private String name;
+
+  @Column(nullable = false)
+  private LocalDate birthday;
+
+  @ManyToMany(cascade = CascadeType.ALL)
+  @JoinTable(name = "book_user",
```

Comment:
The JoinTable annotation is only necessary if you want to re-define the columns of the involved columns and/or the join table name itself.
In this case, you don't need it. And by having the @ManyToMany annotation, it infers automatically that it should construct a JoinTable.